DesignSafe-CI/portal
# Task/des 2418 hazmapper updates

## Overview: ##
Change links on Published for HazMappers

## PR Status: ##


## Related Jira tickets: ##

* [DES-2418](https://jira.tacc.utexas.edu/browse/DES-2418)

## Summary of Changes: ##
1. The folder icon was removed - on all Published HazMappers.
2. Link in Published (open-box w/ arrow icon) goes to .../project-public/...

## Testing Steps: ##
1. Find a project in My Projects, it should have the folder icon. The open-box w/ arrow icon should link to https://hazmapper.tacc.utexas.edu/hazmapper/project/${map.uuid}
3. Find a published project, it should **not** have the folder icon. The open-box w/ arrow icon should link to https://hazmapper.tacc.utexas.edu/hazmapper/project-public/${map.uuid} for PRJ-2277

## UI Photos:

![Screen Shot 2023-03-23 at 4 50 28 PM](https://user-images.githubusercontent.com/35277477/227381447-23dbd681-da62-409b-a9a4-2ef4afd31554.png)

## Notes: ##
